.goreleaser.yml

```diff
@@ -0,0 +1,51 @@
+# This is an example .goreleaser.yml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+env_files:
+  github_token: ~/.config/goreleaser/token
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    # - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      # - windows
+    goarch:
+      - amd64
+    ldflags:
+      - -s -w -X 'github.com/openshift/osdctl/cmd.GitCommit={{.ShortCommit}}'
+
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
```

Comment:
since your `goarch` is `amd64` only then I guess you can drop `386: i386`.
also since `goos` is not using Windows, then you can also drop the `windows: Windows`.

Reply:
Done!